src/main/scala/viper/silver/verifier/ModelParser.scala

```diff
@@ -1,18 +1,24 @@
 package viper.silver.verifier
 
 import fastparse._
-import viper.silver.parser.FastParserCompanion.whitespace
-
 object ModelParser {
+  def modelEntry[_: P]: P[(String, ModelEntry)] = {
+    implicit val ws = viper.silver.parser.FastParserCompanion.whitespaceWithoutNewlineOrComments
+    P(idnuse ~ "->" ~ definition.?).map {
+      case (i, Some(e)) => (i, e)
+      case (i, None) => (i, UnspecifiedEntry)
+    }
+  }
+
+  import viper.silver.parser.FastParserCompanion.whitespace
```

Comment:
Is there a reason why the import was moved from line 4 to here? If not, I would suggest to move it back.